src/screens/LanguageSelect/LanguageSelect.js

```diff
@@ -11,9 +11,24 @@ const LanguageSelect = props => {
       <View style={styles.imageContainer}>
         <Image source={languageBird} resizeMethod="scale" />
         <View style={{}}>
-          <LanguageButton title="Беларуская мова" langKey="be" {...props} />
-          <LanguageButton title="Русский язык" langKey="ru" {...props} />
-          <LanguageButton title="English" langKey="en" {...props} />
+          <LanguageButton
+            title="Беларуская мова"
```

Comment:
please add titles to locales files

Reply:
I most cases I've seen that in choose locale lists locales are written in it's native language
So it's preferable to hold titles as they are

Reply:
I agree with Aliaksei, if apply translate() to the labels, all of them will be translated into one language, and that it not what we have in designs

Reply:
Ok, I got your idea, but I'm speaking from the point of where are we storing labels. We have three separate files for all our labels, but in case above we simply hardcode it. I suggest to use it as `title={translations.en.languageSelect.title}`, for example

Reply:
Discussed with Maryna that makes sense to leave it